store/test-store/README.md

```diff
@@ -0,0 +1,57 @@
+# Setting up for running tests
+
+This document describes what needs to be done to run tests against a
+Postgres database.
```

Comment:
How come the CI setup didn't require any changes?

Reply:
Because running tests the old way with `THEGRAPH_STORE_POSTGRES_DIESEL_URL` and just a primary shard still works; it just bypasses the cross-shard copy tests. I plan on updating the CI config once this is merged, but wanted to document the test setup for others before I do that.